### ISSUE TITLE
Any plan to move plugin to npm

### ISSUE THREAD

**Issue by writeosahon** [VersoSolutions/CordovaClipboard#11]
Hello @VersoSolutions .
Corodova have announced that all plugins should be prepared to move to npm (read http://cordova.apache.org/announcements/2015/04/21/plugins-release-and-move-to-npm.html).

Please, hope you're making plans to move as well? Your plugin is very very important to thousands of developers.

Thanks

**Comment by bau720123** [comment]
+1
according to this
http://phonegap.com/blog/2015/05/26/npm-plugins-available/

**Comment by dcousens** [comment]
+1, @VersoSolutions please move to `npm` as soon as possible :)

**Comment by dcousens** [comment]
bump @VersoSolutions

**Comment by dcousens** [comment]
~~This has [looks like it has] been published as `cordova-plugin-clipboard`.~~

~~@VersoSolutions if you could please update the README, that'd be great.~~

**Comment by dcousens** [comment]
Bump @ctrevisan

**Comment by bau720123** [comment]
hi @dcousens 
a little strange
is the npm version fork by @VersoSolutions ?

because in the npm page
https://www.npmjs.com/package/cordova-plugin-clipboard
it have the "setText" and "getText" method，and seem it is a "BlackBerry" version

but in @VersoSolutions version
https://github.com/VersoSolutions/CordovaClipboard
it use "copy" and paste method，and it have android ios wp8

I think...them are not the same

**Comment by dcousens** [comment]
@bau720123 you are correct,  I just got around to testing this,  and probably shouldn't have mentioned that without actually testing.

**Comment by writeosahon** [comment]
It looks to me like @VersoSolutions have abandoned this repository/plugin because there's been no response from them ever since. If your project is seriously dependent on this plugin, I advise you download / clone it to your desktop and add it to your cordova projects locally.

**Comment by dcousens** [comment]
Shall we fork?

**Comment by lucasfeliciano** [comment]
@dcousens, do you know if some one already forked and published in npm?
If no, I'm looking forward to do so

**Comment by writeosahon** [comment]
@dcousens @lucasfeliciano if we fork, who will be responsible for active maintenance of the plugin??

**Comment by lucasfeliciano** [comment]
I could do that, but the problem is that I know almost nothing about objective c so that might be a problem for maintain it.

**Comment by dcousens** [comment]
@writeosahon I'm able to debug and perform some maintenance,  however going forward as things change,  its likely I would need contributor support for devices/platforms I can't test on.

**Comment by writeosahon** [comment]
@lucasfeliciano @dcousens ok guys!! in that case, go ahead and fork. Please leave a comment with a link to the fork in this thread so people interested can easily locate your fork and use it.

Thanks guys

**Comment by AllSeeingEye** [comment]
I've published to NPM as "cordova-universal-clipboard":

https://www.npmjs.com/package/cordova-universal-clipboard

Please make pull requests to the repo:

https://github.com/Visigo/CordovaClipboard

where we'll try to maintain the plugin.

**Comment by bau720123** [comment]
thanks @AllSeeingEye
